### PR TITLE
DLCQuest: Use options API for campaign and remove unused imports in tests

### PR DESCRIPTION
**Pull request by nicholassaylor**
## What is this fixing or adding?
Removes unused imports in DLCQuest world tests and uses Options API for checking the value of campaign

## How was this tested?
Running unittests

### CODE REVIEW

**Comment by nicholassaylor** [comment]
@axe-y and/or @agilbert1412 , requesting your review

**Review by Exempt-Medic — APPROVED** [review]
The code LGTM and the tests obviously pass.

File: worlds/dlcquest/test/checks/world_checks.py [hunk 1]
```diff
@@ -39,4 +39,4 @@ def assert_can_win(tester: DLCQuestTestBase, multiworld: MultiWorld):
 
 def assert_same_number_items_locations(tester: DLCQuestTestBase, multiworld: MultiWorld):
     non_event_locations = [location for location in multiworld.get_locations() if not location.advancement]
-    tester.assertEqual(len(multiworld.itempool), len(non_event_locations))
```
Comment (agilbert1412) [inline worlds/dlcquest/test/checks/world_checks.py:42]: The IDE generally recommends keeping the empty line at the end of the file

**Review by agilbert1412 — APPROVED** [review]

File: worlds/dlcquest/test/checks/world_checks.py (hunk 1 above)
Comment (Exempt-Medic) [reply worlds/dlcquest/test/checks/world_checks.py:42]: This is adding a blank line, which is recommended

Comment (agilbert1412) [reply worlds/dlcquest/test/checks/world_checks.py:42]: Oh, is it? Maybe I misunderstood the git diff, it looked to me like the change was removing the line. Carry on then.

**Review by axe-y — APPROVED** [review]

**Review by NewSoupVi — APPROVED** [review]

**Pull request merged**